src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -111,12 +119,17 @@
             <BaseAddress class="peoples-roles-mailing-address" :address="orgPerson.mailingAddress" />
           </v-col>
 
-          <!-- Delivery Address (for persons only) -->
+          <!-- Delivery Address -->
           <v-col cols="12" sm="3" :class="{ 'removed': wasRemoved(orgPerson)}">
-            <p v-if="isSame(orgPerson.mailingAddress, orgPerson.deliveryAddress, ['id'])"
-              class="peoples-roles-delivery-address info-text">Same as Mailing Address
-            </p>
-            <BaseAddress v-else class="peoples-roles-delivery-address" :address="orgPerson.deliveryAddress"/>
+            <template v-if="hasRoleDirector(orgPerson) || hasRoleProprietor(orgPerson) || hasRolePartner(orgPerson)">
```

Comment:
Don't display the delivery address unless this is a director, proprietor or partner, eg,

![image](https://user-images.githubusercontent.com/10002889/171968404-4462298f-01b8-4ab4-89ca-43f30b3613ba.png)